scalameta/parsers/js/src/main/scala/scala/meta/parsers/JSFacade.scala

```diff
@@ -59,7 +61,12 @@ object JSFacade {
       case _ => js.Dynamic.literal()
     }
```

Comment:
Actually, I just realized that `.productFields` will only solve what *keys* to put into the dictionary, not the values. 
I think we can remove the FIXME comment and claim this as a final solution. It might be good to handle the custom cases at the top and then fallback with a `case Lit(value) => value`, which will do the right thing for int/double/float/...